test/utils.js

```diff
@@ -36,11 +36,28 @@ export function almostEqual(a, b, criteria) {
   }
 }
 
-export function checkValue(
-    output, expected, criteria = opFp32AccuracyCriteria) {
+export function compareUlp(a, b, nulp = 1, base = 64) {
+  const aBuffer = new ArrayBuffer(8);
+  const bBuffer = new ArrayBuffer(8);
+  const aView = new DataView(aBuffer);
```

Comment:
It should populate the buffer with 0 or ~0, as C++ code does
```c++
int64_t bitwiseValue = (value < T(0)) ? ~int64_t(0) : 0; // Extend sign.
```

You probably can use a JavaScript `BigInt` object do that e.g., 
```js
aView.setBigInt64(0, value < 0 ? ~BigInt(0) : BigInt(0))
```